client/src/pages/home/home.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import logo from '../../logo.svg';
+import './home.css'; //import css for this page
+function Home() {   {/* This page is the exact same page as the original example */}
+  return (
+    <div className="Home">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
```

Comment:
This might be personal preference, but I find multiline ```a``` tags hurt readability.

Reply:
Actually this is a thing in React, because of how many props get passed through tags

`I dislike it also but is a legal and standard way of doing it`

Reply:
I just copy and pasted the original App.js content into this page.

Reply:
Multiline tags are pretty standard in production development as usually it is easier to read. Thought @Mutilar has other opinions